src/components/layout/Nav.module.scss

```
@@ -0,0 +1,58 @@
+@use "../../styles/abstracts/variables" as *;
+@use "../../styles/abstracts/container";
+
+.nav {
+  position: sticky;
+  top: 0;
+  z-index: 10;
+  // 스크롤 시 콘텐츠가 배경으로 비치도록 반투명 처리.
```

Stylelint (17.14.0)

[error] Expected empty line before comment (scss/double-slash-comment-empty-line-before)

(scss/double-slash-comment-empty-line-before)